source/patterns/01-molecules/contextual-data/contextual-data.twig

```diff
@@ -0,0 +1,3 @@
+<div class="contextual-data">
```

Comment:
- As discussed, need to set attributes at the root level here as we do for the other patterns.
- Needs a `with` and a `block` wrapping the include.